data/json/mapgen/house18.json

```diff
@@ -0,0 +1,147 @@
+                "+": "t_door_c",
+                "2": "t_window_domestic",
+                "3": "t_door_locked",
+                "l": "t_linoleum_gray",
+                "s": "t_sidewalk",
+                "t": "t_linoleum_gray",
+                "6": "t_floor",
+                "7": "t_floor",
+                "8": "t_linoleum_gray",
+                "9": "t_floor",
+                ":": "t_floor",
+                ">": "t_floor",
+                "?": "t_floor",
+                "@": "t_floor",
+                "^": "t_floor",
+                "A": "t_floor",
+                "B": "t_floor",
+                "C": "t_floor",
+                "E": "t_floor",
+                "F": "t_floor",
+                "G": "t_floor",
+                "S": "t_linoleum_gray",
+                "L": "t_floor"
+            },
+            "furniture": {
+                "6": "f_table",
+                "7": "f_bookcase",
+                "8": "f_bathtub",
+                "9": "f_rack",
+                ":": "f_dresser",
+                ">": "f_counter",
+                "?": "f_sofa",
+                "@": "f_bed",
+                "^": "f_indoor_plant",
+                "A": "f_sink",
+                "B": "f_chair",
+                "C": "f_desk",
+                "E": "f_cupboard",
+                "F": "f_fridge",
+                "G": "f_oven",
+                "S": "f_sink",
+                "L": "f_locker"
+            },
+            "toilets": {
+                "t": { }
+            },
+            "comment": "specific terrain/furniture/radiation. doing this until grass_or_dirt is supported ---v",
+            "set": [
+                { "point": "terrain", "id": "t_dirt", "x": 0,       "y": [1,23], "repeat": [5,10] },
+                { "point": "terrain", "id": "t_dirt", "x": [22,23], "y": [1,23], "repeat": [10,20] }
+            ],
+            "comment": "randomly picked from a group ---v",
+            "place_loot": [
+                { "group": "homebooks",   "chance": 70, "x": [3,3],   "y": [16,16], "repeat": [1,2] },
+                { "group": "homebooks",   "chance": 70, "x": [5,5],   "y": [16,16], "repeat": [1,2] },
+                { "group": "homebooks",   "chance": 70, "x": [16,17], "y": [16,16], "repeat": [1,4] },
+                { "group": "magazines",   "chance": 80, "x": [4,5],   "y": [4,5],   "repeat": [1,3] },
+                { "group": "magazines",   "chance": 90, "x": [16,17], "y": [2,2],   "repeat": [1,5] },
+                { "group": "novels",      "chance": 90, "x": [10,10], "y": [5,5],   "repeat": [1,4] },
+                { "group": "office",      "chance": 80, "x": [17,19], "y": [22,22], "repeat": [1,4] },
+                { "group": "livingroom",  "chance": 90, "x": [2,6],   "y": [3,8],   "repeat": [1,8] },
+                { "group": "dining",      "chance": 85, "x": [17,19], "y": [5,6],   "repeat": [1,4] },
+                { "group": "dining",      "chance": 70, "x": [4,4],   "y": [14,14], "repeat": [1,8] },
```

Comment:
> It doesn't mater here since dining contains useless stuff.

I'm not sure whether that affects whether you should spawn it. You have no control over what may be added to the `dining item group in future

> If we decrease repeat house will be poor on loot.

There is already too much (and poorly placed) loot. Incremental improvement in this situation would be of great benefit.

> Actually I might decrease chance instead 5-10% overall.

It would be better to balance `repeat` as this is probably where scaling will later take place.